common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1371,12 +1397,24 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
 
 object CelebornConf extends Logging {
 
-  val TRANSPORT_MODULE_FALLBACKS = Map(
-    "rpc_service" -> "rpc",
-    "rpc_app" -> "rpc",
+  val TRANSPORT_MODULE_FALLBACKS: Map[String, String] = Map(
```

Comment:
Would this fallback document later?

Reply:
Yes, this will be documented as part documenting the SSL feature.
See [this comment](https://issues.apache.org/jira/browse/CELEBORN-1353?focusedCommentId=17837988&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17837988) in particular to ensure it is not missed.

Reply:
@mridulm, do the fallbacks have any test case to verify?

Reply:
Yes, these were added in previous PR btw.
See [here](https://github.com/apache/celeborn/blob/11c7d1080c76b86061b46ae3f99ceade37086f82/common/src/test/scala/org/apache/celeborn/common/CelebornConfSuite.scala#L379) for example.

In the current PR as well, the fallbacks are exercised - the configs are set as users would set them (so not for lifecycle manager, as that is internal), while the test itself would be for the internal module - and so the configs resolution would exercise fallback.